### PR TITLE
[MRG] Fix `phy cluster-auto`  command

### LINKED ISSUE #485: phy cluster-auto should take a PRM file

### PR DESCRIPTION
**Pull request by rossant**
Closes #485.

### CODE REVIEW

**Comment by rossant** [comment]
status: waiting to fix KK2 split problem with callback

**Comment by rossant** [comment]
ready to go @nippoo

File: phy/scripts/phy_script.py [hunk 1]
```diff
@@ -294,11 +294,11 @@ def detect(args):
 
 
 def cluster_auto(args):
+    assert args.file.endswith('.kwik')
```
Comment (nippoo) [inline phy/scripts/phy_script.py]: `assert`? this should probably be in `_create_session`, no? And also fail with a more sensible error than just an `AssertError`?

I can imagine some people renaming their .prm to a .kwik ;-)

Comment (rossant) [inline phy/scripts/phy_script.py]: yeah... the `phy_script` module needs a refactoring too...

**Pull request merged**